clientd3d/d3drender.c

```diff
@@ -7131,14 +7143,15 @@ void D3DRenderObjectsDraw(d3d_render_pool_new *pPool, room_type *room,
 				pChunk->st1[3].s = D3DRENDER_CLIP_TO_SCREEN_X(topLeft.x, gScreenWidth) / gScreenWidth;
 				pChunk->st1[3].t = D3DRENDER_CLIP_TO_SCREEN_Y(topLeft.y, gScreenHeight) / gScreenHeight;
 
-				pChunk->st1[0].s -= (gFrame & 3) / 256.0f;
-				pChunk->st1[0].t -= (gFrame & 3) / 256.0f;
-				pChunk->st1[1].s -= (gFrame & 3) / 256.0f;
-				pChunk->st1[1].t += (gFrame & 3) / 256.0f;
-				pChunk->st1[2].s += (gFrame & 3) / 256.0f;
-				pChunk->st1[2].t += (gFrame & 3) / 256.0f;
-				pChunk->st1[3].s += (gFrame & 3) / 256.0f;
-				pChunk->st1[3].t -= (gFrame & 3) / 256.0f;
+				float animationFactor = 2560.0f;

```

Comment:
This one is 2560, but all the rest are 256?

Does this constant actually have to do with animation?

Some comments seem necessary here.